### DISCUSSION TITLE
Welcome to programmer Discussions!

### DISCUSSION THREAD

**Discussion by nydevel** [nydevel/programmer#9] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DagothMor** [comment]
Junior Backend по шарпу,начал смотреть Ясинского с самого зарождения(еще до огромной популярности видоса про вышку)
Максимум исправлял опечатки в блоке Николая.
@nydevel написал бы для чего дискуссию решил открыть)Просто на вопросы ответить возникающие при чтении твоего блога?

**Reply by nydevel** [reply]
Привет, да, все верно, плюс протестировать функционал дискуссий на гитхабе
